comp/core/workloadmeta/filter_test.go

```diff
@@ -0,0 +1,350 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-present Datadog, Inc.
+
+package workloadmeta
+
+import (
+	"fmt"
+	"slices"
+	"testing"
+
+	"gotest.tools/assert"
```

Comment:
We normally use `github.com/stretchr/testify/assert` instead of this.

Reply:
Yes, sorry was added by the code editor 🥲 

Fixing it..